examples/AcousticWave2D_codegen.py

```diff
@@ -78,14 +78,21 @@ def damp_boundary(damp):
         self.dm = dm
 
     def Forward(self):
-        fw = ForwardOperator(self.m, self.src, self.damp, self.rec, self.u, time_order=self.t_order, spc_order=self.s_order, cache_blocking=True, profile=True, auto_tune=True)
+        fw = ForwardOperator(self.m, self.src, self.damp, self.rec, self.u, time_order=self.t_order, spc_order=self.s_order, cache_blocking=True, profile=True)
         fw.apply()
-        return (self.rec.data, self.u.data)
+        return self.rec.data, self.u.data
+
+    # Forward with auto tuning enabled
+    def Forward_auto_tune(self):
```

Comment:
I had an assumption that travis runs benchmark. Going to fix it. However should I still have separate file for at_benchmark or should I just intergrate it to normal benchmark (it does have a lot of stuff that I don't need though)